src/mac-media-keys/meson.build

```diff
@@ -9,6 +9,7 @@ shared_module('mac-media-keys',
   mac_media_keys_sources,
   dependencies: [audacious_dep],
   name_prefix: '',
+  objc_args: ['-std=c99', '-fobjc-exceptions'],
```

Comment:
Is it really necessary to specify the c99 std? What is the error message with GCC? 

And for consistency I think it's better to update `mac-media-keys/Makefile` for Autotools as well.